CONTRIBUTING.md

```diff
@@ -0,0 +1,402 @@
+Instead, we recommend using [Logseq forum][forum] to ask support-related questions. 
+
+The Logseq forum is a much better place to ask questions since:
+
+- there are more people willing to help on the forum
+- questions and answers stay available for public viewing so your question/answer might help someone else
+- The forum's voting system assures that the best answers are prominently visible.
+
+To save your and our time, we will systematically close all issues that are requests for general support and redirect people to the forum.
+
+If you would like to chat about the question in real-time, you can reach out via [our Discord server][discord].
+
+
+## <a name="issue"></a> Found a Bug?
+
+If you find a bug in the source code, you can help us by [submitting an issue](#submit-issue) to our [GitHub Repository][github].
+Even better, you can [submit a Pull Request](#submit-pr) with a fix.
+
+
+## <a name="feature"></a> Missing a Feature?
+You can *request* a new feature by [Creating a thread][feature-request] in our forum.
+If you would like to *implement* a new feature, please consider the size of the change in order to determine the right steps to proceed:
+
+* For a **Major Feature**, first open an issue and outline your proposal so that it can be discussed.
+  This process allows us to better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
+
+  **Note**: Adding a new topic to the documentation, or significantly re-writing a topic, counts as a major feature.
+
+* **Small Features** can be crafted and directly [submitted as a Pull Request](#submit-pr).
+
+
+## <a name="submit"></a> Submission Guidelines
+
+
+### <a name="submit-issue"></a> Submitting an Issue
+
+Before you submit an issue, please search the [issue tracker][issue-tracker]. An issue for your problem might already exist and the discussion might inform you of workarounds readily available.
+
+Please include the following with each issue:
+
+- Version of Logseq
+- Your operating system
+- List of extensions that you have installed. Attempt to recreate the issue after disabling all extensions.
+- Reproducible steps (1... 2... 3...) that cause the issue
+- What you expected to see, versus what you actually saw
+- Images, animations, or a link to a video showing the issue occurring
+- A code snippet that demonstrates the issue or a link to a  code repository the developers can easily pull down to recreate the  issue locally
+  - **Note:** Because the developers need to copy and paste the code snippet, including a code snippet as a media file (i.e. .gif)  is not sufficient.
+- Errors from the Dev Tools Console (open from the menu: View > Toggle Developer Tools or press CTRL + Shift + i)
+
+You can file new issues by selecting from our [new issue templates][new-issue] and filling out the issue template.
```

Comment:
@logseq-cldwalker don't you think that's a better fit for the docs folder? 
I do think such section will add value; We can also link to it from the issue template.
The only reason I am hesitant to include it in this document is that it will be long and will require screenshots etc..